CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ### Master
 
+### 0.20.4
+- Rename redis resources ([#318](https://github.com/shopify/kubernetes-deploy/pull/318))
+- Don't consider pod preempting a failure ([#317](https://github.com/shopify/kubernetes-deploy/pull/317))
```

Comment:
Sorry if that comment wasn't clear. I think this line is the only one that should be in the (public) changelog.